### ISSUE TITLE
react framework

### ISSUE THREAD

**Issue by jeongdae1225** [tjwndud/Last-PJ#2]
순수 html 파일로 개발 중인 듯 보입니다. React 같은 framework 이용할 계획이 있을까요?

**Comment by tjwndud** [comment]
네 지금은 프론트 자체가 처음이라 html/css/js를 공부하면서 구현해보았고 지금은 js문법과 리엑트를 공부중에 있습니다 
따라서 지금처럼 공부하면서 구현하는 방향으로 진행할 것 같습니다.

**Comment by jeongdae1225** [comment]
공부중 도움이 필요하면 언제든 issue 남겨주세요.
